site/docs/api/Editor.md

```diff
@@ -100,4 +101,64 @@ const App = () => {
     </Editor>
   )
 }
+```
+
+
+### Override default event handlers
+Customize how the default event handlers are handled
+
+```tsx {9-40,46-48}
+import {
+  DefaultEventHandlers,
+  DefaultEventHandlersOptions,
+  Editor,
+  EditorStore,
+  NodeId
+} from '@craftjs/core'
+
+class CustomEventHandlers extends DefaultEventHandlers {
+  isMultiSelectEnabled: (e: MouseEvent) => boolean
+
+  constructor(options: DefaultEventHandlersOptions & { store: EditorStore }) {
+    super(options)
+    this.isMultiSelectEnabled = options.isMultiSelectEnabled
+  }
+
+  handlers() {
+    const defaultHandlers = super.handlers()
+
+    return {
+      ...defaultHandlers,
+      // Customize the hover event handler
+      hover: (el: HTMLElement, id: NodeId) => {
+        const unbindMouseover = defaultHandlers.hover(el, id)
```

Comment:
Perhaps it's better to name this variable `unbindDefaultHoverHandler` since the internal `hover` could technically contain events other than just `mouseover`